homeassistant/components/hue/v2/light.py

```diff
@@ -89,6 +89,7 @@ def __init__(
                 self._supported_color_modes.add(ColorMode.BRIGHTNESS)
             # support transition if brightness control
             self._attr_supported_features |= LightEntityFeature.TRANSITION
+        self._restore_brightness: float | None = None
```

Comment:
maybe "_last_brightness" is a better name for this variable as we're string the last known brightness in this variable here

Reply:
Changed it, thanks for the suggestion